docs/source/overview.rst

```diff
@@ -29,7 +29,9 @@ CuPy also includes the following features for performance:
 
 - User-defined elementwise CUDA kernels
 - User-defined reduction CUDA kernels
+- Just-in-time compiler converting Python functions to CUDA kernels
```

Comment:
cc: @asi1024 I think `@cupyx.jit` should be mentioned in the Overview page, but not sure if this is the best place.

Reply:
I think it is appropriate place! Thanks!